Features/Transfer/Sources/ViewModels/AmountSceneViewModel.swift

```diff
@@ -309,6 +310,20 @@ extension AmountSceneViewModel {
         }
     }
 
+    private func getRecipientAddress(chain: StakeChain?, type: AmountType, validatorId: String?) -> String? {
+        guard let id = validatorId else {
```

Comment:
Move this guard into the specific case, because for smartChain it can cause an early return